features/step_definitions/networking.rb

```diff
@@ -641,3 +641,16 @@
 
   raise "Multus is not running correctly!" unless desired_multus_replicas == available_multus_replicas && available_multus_replicas != 0
 end
+
+
+Given /^I run network command on the#{OPT_QUOTED} node's sdn pod:$/ do |node_name, table|
```

Comment:
I think we can skip word `network` from step name because I see you can run any command. Step does not seems pecific to network commands. You can happily run `ls`, `cat` or whatever from what I see.

Reply:
It is ok to use which step name for me. Just in this case, only network commands make sense since the sdn pod is using hostNetwork, means network related commands run in the pod are same as run them on the node.